converter/text.py

```diff
@@ -257,13 +262,13 @@ def kerning(fig_text):
     if "letterSpacing" in fig_text:
         match fig_text["letterSpacing"]:
             case {"units": "PIXELS", "value": pixels}:
-                return pixels
+                return pixels if pixels != 0 else None
```

Comment:
A comment here might be nice, saying we convert kerning = 0 to kerning = Auto (None) since it matches better.

Reply:
https://github.com/sketch-hq/fig2sketch/pull/58/commits/12226642874dd829eaad2b8f271624e10562baaa